PF/Assignment1/MenuClass.java

```diff
@@ -0,0 +1,13 @@
+
+public class MenuClass {
+
+	public static void showBasicMenu() {
```

Comment:
ACTION: It is not good to create a new class to show Menu. Please remove this class and move this function in MainSocialNetwork  class.